apps/fxc-front/src/app/logic/score/scorer.ts

```diff
@@ -17,124 +13,15 @@ export class Score {
   closingRadius: number | null;
   points: number;
 
-  constructor(score: Partial<Score>) {
+  constructor(score: Partial<Omit<Score,'points'>>) {
```

Comment:
`Partial<Omit<Score,'points'>>` looks convoluted - that's was true before your PR.

Can you add a comment about the rationale.

If I had to do this today I would probably not use the constructor but a `initFrom` method (the name is just an example, could be anything that makes sense and maybe explain the Partial/Omit)

Reply:
removed the logic for `points` in constructor and then the code becomes simpler: No `Partial` with `Omit`, no mutation method,...